app/assets/stylesheets/helpers/_header.scss

```diff
@@ -341,3 +341,47 @@
     margin-right: .5em;
   }
 }
+
+
+.show-global-bar #global-header-bar {
+  display: none;
+}
+
+#global-bar {
+  margin-bottom: 15px;
```

Comment:
We should probably be using some mixins for the spacing - I think you can get 15px with `govuk-spacing(3)` - https://design-system.service.gov.uk/styles/spacing/

Reply:
We're not going to address this immediately because this helper file contains a very mixed bag of styling via variables and pixel values. The reasoning is we're either going to spend the time maturing this banner code into something more permanent or we'll remove after understanding user engagement better.